map_model/src/edits/mod.rs

```diff
@@ -77,13 +73,17 @@ impl EditRoad {
     fn diff(&self, other: &EditRoad) -> Vec<String> {
         let mut lt = 0;
         let mut dir = 0;
-        for ((lt1, dir1), (lt2, dir2)) in self.lanes_ltr.iter().zip(other.lanes_ltr.iter()) {
-            if lt1 != lt2 {
+        let mut width = 0;
+        for (spec1, spec2) in self.lanes_ltr.iter().zip(other.lanes_ltr.iter()) {
+            if spec1.lt != spec2.lt {
                 lt += 1;
             }
-            if dir1 != dir2 {
+            if spec1.dir != spec2.dir {
                 dir += 1;
             }
+            if spec1.width != spec2.width {
+                width += 1;
+            }
         }
```

Comment:
In terms of metrics, "road width" might an interesting thing to track - e.g. if we split a driving lane into two cycle lanes such that the total road width doesn't change, that's a different value proposition from adding a cycle lane which requires the road to be widened.

Reply:
Not necessary for this PR, just food for thought.